webapp/store/actions/navActions.js

```diff
@@ -33,7 +33,7 @@ export function loadSideNav() {
     let sideNav = sortedNavItems({ pluginMetadata });
     // add unique id to each nav item
     sideNav = sideNav.map(item => {
-      item.id = item.id ? item.id : helpers.getHash(item).slice(0, 6);
+      item.id = item.id ? item.id : helpers.getHash(item, 'sha256', 0, 6);
```

Comment:
By slicing the hash, you are increasing the probability of a collision

Reply:
Yeah, since this is really just as a convenience, something added based on our previous discussions, and you won't really ever have more than 1-2 dozen side nav items, the chance of a collision seemed low enough that it wasn't worth doing extra passes to resolve.

Reply:
👍